edx_django_utils/monitoring/tests/code_owner/test_middleware.py

```diff
@@ -53,28 +62,38 @@ def test_request_call(self):
 
     @override_settings(
         CODE_OWNER_MAPPINGS={'team-red': ['edx_django_utils.monitoring.tests.code_owner.mock_views']},
+        CODE_OWNER_THEMES={'team': ['team-red']},
         ROOT_URLCONF=__name__,
     )
-    @patch('edx_django_utils.monitoring.internal.code_owner.middleware.set_custom_attribute')
+    @patch(
+        'edx_django_utils.monitoring.internal.code_owner.utils.set_custom_attribute',
+        new_callable=get_set_custom_attribute_mock
+    )
+    @patch(
+        'edx_django_utils.monitoring.internal.code_owner.middleware.set_custom_attribute',
+        new_callable=get_set_custom_attribute_mock
+    )
```

Comment:
**Note for Reviewer**: This is ugly, but is it worth trying to create a decorator that wraps this? Not sure how easy or hard that will be.

Reply:
We're sticking with this. It works.